com.microsoft.EdgeDev.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 the Chromium authors, with modifications -->
+<component type="desktop">
+  <id>com.microsoft.EdgeDev</id>
+  <launchable type="desktop-id">com.microsoft.EdgeDev.desktop</launchable>
+  <name>Microsoft Edge (dev)</name>
```

Comment:
Can you make this "Microsoft Edge Dev" instead, also in the desktop file? Seems that's what MS uses on other channels.

Reply:
It's what they use in their desktop file in the deb package. The flathub beta package is also called Microsoft Edge (beta). Do you think I should change it anyway?